content/docs/introduction/billing-azure-marketplace.md

```diff
@@ -0,0 +1,61 @@
+
+## Prerequisites for setting up the Neon integration
+
+- **Azure account**: If you don't have an active Azure subscription, [create a free account](https://azure.microsoft.com/free).
+- **Access level**: Only users with **Owner** or **Contributor** access on the Azure subscription can set up the Azure integration. Ensure you have the appropriate access before proceeding.
+
+## Find Neon on Azure and subscribe
+
+1. Use the search in the [Azure portal](https://portal.azure.com/) to find the **Neon Serverless Postgres** offering.
+2. Alternatively, go to the [Azure Marketplace](https://portal.azure.com/#view/Microsoft_Azure_Marketplace/MarketplaceOffersBlade/selectedMenuItemId/home) and search for **Neon Serverless Postgres**.
+3. Subscribe to the service.
+
+After subscribing, you will be directed to the [Create a Neon Resource](#create-a-neon-resource) page.
+
+## Create a Neon Resource
+
+1. On the **Create a Neon Resource** page, set the following values in the **Create Neon Resource** section.
+
+   | Property                   | Description                                                                                                                                                                                                                                                                                                        |
+   | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+   | **Subscription**           | From the drop-down, select your Azure subscription where you have Owner or Contributor access.                                                                                                                                                                                                                     |
+   | **Resource group**         | Specify whether you want to create a new resource group or use an existing one. A resource group is a container that holds related resources for an Azure solution. For more information, see [Azure Resource Group overview](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/overview). |
+   | **Resource Name**          | Enter the name for the Neon organization you want to create.                                                                                                                                                                                                                                                       |
```

Comment:
Can we make this clearer that it's the Azure resource name? This field will only be used on the Azure side.